### PR TITLE
285 review all modules name prior to 20 release

### LINKED ISSUE #285: Review all Modules name prior to 2.0 release
### Description

Review and update any names that need to be updated.

In the Change Log we need to have a **breaking changes ** section for every name we change. 
Similar to how AWS does this (https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md#500-may-25-2023) 

### References

_No response_

### Would you like to implement a fix?

None

### PR DESCRIPTION
**Pull request by wenjun666**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,115 @@
...
+* **Rename Data Source:** `netapp-ontap_cluster_licensing_license_data_source` is now renamed to `netapp-ontap_cluster_licensing_license`
+* **Rename Data Source:** `netapp-ontap_cluster_licensing_licenses_data_source` is now renamed to `netapp-ontap_cluster_licensing_licenses`
+* **Rename Data Source:** `netapp-ontap_cluster_peer_data_source` is now renamed to `netapp-ontap_cluster_peer`
+* **Rename Data Source:** `netapp-ontap_cluster_peers_data_source` is now renamed to `netapp-ontap_cluster_peers`
+* **Rename Data Source:** `netapp-ontap_cluster_schedule_data_source` is now renamed to `netapp-ontap_cluster_schedule`
+* **Rename Data Source:** `netapp-ontap_cluster_schedules_data_source` is now renamed to `netapp-ontap_cluster_schedules`
+
+Name Services
+* **Rename Resource:** `netapp-ontap_name_services_dns_resource` is now renamed to `netapp-ontap_dns`
+* **Rename Resource:** `netapp-ontap_name_services_ldap_resource` is now renamed to `netapp-ontap_name_services_ldap`
```
Comment (chuyich) [inline CHANGELOG.md:18]: Just curious, why the dns does not need the prefix "name_services" but ldap needs it?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,115 @@
...
+Networking
+* **Rename Resource:** `netapp-ontap_networking_ip_interface_resource` is now renamed to `netapp-ontap_network_ip_interface`
+* **Rename Resource:** `netapp-ontap_networking_ip_route_resource` is now renamed to `netapp-ontap_network_ip_route`
+* **Rename Data Source:** `netapp-ontap_networking_ip_interface_data_source` is now renamed to `netapp-ontap_network_ip_interface`
+* **Rename Data Source:** `netapp-ontap_networking_ip_interfaces_data_source` is now renamed to `netapp-ontap_network_ip_interfaces`
+* **Rename Data Source:** `netapp-ontap_networking_ip_route_data_source` is now renamed to `netapp-ontap_network_ip_route`
+* **Rename Data Source:** `netapp-ontap_networking_ip_routes_data_source` is now renamed to `netapp-ontap_network_ip_routes`
+
+Protocols
+* **Rename Resource:** `netapp-ontap_cifs_local_group_member_resource` is now renamed to `netapp-ontap_cifs_local_group_members`
```
Comment (chuyich) [inline CHANGELOG.md:33]: Should "member" or "members"?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,115 @@
...
+* **Rename Resource:** `netapp-ontap_cifs_local_group_resource` is now renamed to `netapp-ontap_cifs_local_group`
+* **Rename Resource:** `netapp-ontap_cifs_local_user_resource` is now renamed to `netapp-ontap_cifs_local_user`
+* **Rename Resource:** `netapp-ontap_cifs_service_resource` is now renamed to `netapp-ontap_cifs_service`
+* **Rename Resource:** `netapp-ontap_cifs_share_resource` is now renamed to `netapp-ontap_cifs_share`
+* **Rename Resource:** `netapp-ontap_cifs_user_group_privilege_resource` is now renamed to `netapp-ontap_cifs_user_group_privileges`
```
Comment (chuyich) [inline CHANGELOG.md:38]: Should "privileges" or "privilege"?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,8 +1,115 @@
...
+* **Rename Resource:** `netapp-ontap_nfs_export_policy_resource` is now renamed to `netapp-ontap_nfs_export_policy`
+* **Rename Resource:** `netapp-ontap_nfs_export_policy_rule_resource` is now renamed to `netapp-ontap_nfs_export_policy_rule`
+* **Rename Resource:** `netapp-ontap_nfs_service_resource` is now renamed to `netapp-ontap_nfs_service`
+* **Rename Resource:** `netapp-ontap_san_igroup_resource` is now renamed to `netapp-ontap_san_igroup`
+* **Rename Resource:** `netapp-ontap_san_lun-maps_resource` is now renamed to `netapp-ontap_san_lun-map`
```
Comment (chuyich) [inline CHANGELOG.md:43]: Why use "-" not "_"?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,8 +1,115 @@
...
+* **Rename Data Source:** `netapp-ontap_nfs_export_policy_data_source` is now renamed to `netapp-ontap_nfs_export_policy`
+* **Rename Data Source:** `netapp-ontap_nfs_export_policies_data_source` is now renamed to `netapp-ontap_nfs_export_policies`
+* **Rename Data Source:** `netapp-ontap_nfs_export_policy_rule_data_source` is now renamed to `netapp-ontap_nfs_export_policy_rule`
+* **Rename Data Source:** `netapp-ontap_nfs_export_policy_rules_data_source` is now renamed to `netapp-ontap_nfs_export_policy_rules`
+* **Rename Data Source:** `netapp-ontap_nfs_service_data_source` is now renamed to `netapp-ontap_nfs_service`
+* **Rename Data Source:** `netapp-ontap_nfs_services_data_source` is now renamed to `netapp-ontap_nfs_services`
+* **Rename Data Source:** `netapp-ontap_san_igroup_data_source` is now renamed to `netapp-ontap_san_igroup`
+* **Rename Data Source:** `netapp-ontap_san_igroups_data_source` is now renamed to `netapp-ontap_san_igroups`
+* **Rename Data Source:** `netapp-ontap_san_lun-map_data_source` is now renamed to `netapp-ontap_san_lun-map`
```
Comment (chuyich) [inline CHANGELOG.md:65]: "-" vs "_"?

**Review by chuyich — COMMENTED** [review]
Please take a look and answer my questions.

File: CHANGELOG.md (hunk 1 above)
Comment (wenjun666) [reply CHANGELOG.md:18]: This is how I determine: In the VSIM, if you type dns, it knows exactly what dns. If you type ldap, it's ambiguous. Also in the API page, there are multiple ldap.

File: CHANGELOG.md (hunk 2 above)
Comment (wenjun666) [reply CHANGELOG.md:33]: "members", i check the API, you can create a group of members at one time. We haven't implemented it, but it is there.

File: CHANGELOG.md (hunk 3 above)
Comment (wenjun666) [reply CHANGELOG.md:38]: same as above, you can create a list of privileges at one time.

**Review by carchi8py — APPROVED** [review]

**Pull request merged**